### PR TITLE
[go/zh-cn] Add translation

### PR DESCRIPTION
**Pull request by 180909**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: zh-cn/go-cn.html.markdown [hunk 1]
```diff
@@ -141,7 +141,7 @@ func learnTypes() {
 // 只用写return一个词就能将函数内指定名称的变量返回
 func learnNamedReturns(x, y int) (z int) {
     z = x * y
-    return // z is implicit here, because we named it earlier.
+    return // z是隐式的，因为我们已经在前面命名了他。
```
Comment (imba-tjd) [inline zh-cn/go-cn.html.markdown:144]: 我感觉这句翻译的不太好，但是我也给不出更好的翻译……

Comment (lzw-723) [reply zh-cn/go-cn.html.markdown:144]: `在前面已经声明了z，所以在这里是隐式的。`
这样有没有更好？

Comment (imba-tjd) [reply zh-cn/go-cn.html.markdown:144]: 我想翻译成`隐式返回z，因为前面指定了它`

**Comment by imba-tjd** [comment]
Thanks for your contribution!

**Pull request merged**